tests/test_booster/test_plugin/test_gemini_plugin.py

```diff
@@ -26,14 +26,12 @@ def check_gemini_plugin(early_stop: bool = True):
             continue
         # These models are not compatible with gemini
         if name in [
-                'diffusers_clip_vision_model', 'timm_resnet', 'timm_beit', 'timm_beitv2', 'timm_eca_nfnet',
-                'timm_efficientformer', 'timm_hrnet_w18_small', 'timm_nf_ecaresnet101', 'timm_nf_regnet_b0',
-                'timm_skresnet18', 'timm_wide_resnet50_2', 'timm_convit', 'timm_dm_nfnet', 'timm_swin_transformer',
+                'diffusers_clip_vision_model', 'timm_beit', 'timm_beitv2', 'timm_eca_nfnet', 'timm_efficientformer',
+                'timm_nf_ecaresnet101', 'timm_nf_regnet_b0', 'timm_convit', 'timm_dm_nfnet', 'timm_swin_transformer',
                 'torchaudio_conformer', 'torchaudio_deepspeech', 'torchaudio_wavernn', 'torchaudio_tacotron',
                 'deepfm_interactionarch', 'deepfm_simpledeepfmnn', 'dlrm', 'dlrm_interactionarch',
-                'torchvision_googlenet', 'torchvision_inception_v3', 'torchvision_mobilenet_v3_small',
-                'torchvision_resnet18', 'torchvision_resnext50_32x4d', 'torchvision_wide_resnet50_2',
-                'torchvision_vit_b_16', 'torchvision_convnext_base', 'torchvision_swin_s', 'transformers_albert',
+                'torchvision_googlenet', 'torchvision_inception_v3', 'torchvision_vit_b_16',
+                'torchvision_convnext_base', 'torchvision_swin_s', 'transformers_albert',
```

Comment:
Some models have been removed? Why?

Reply:
The list of models don't work with gemini, now some of they work well because inplace operation is supported, so I remove them from the list.

Reply:
That makes sense.